src/compas_fab/robots/robot.py

```diff
@@ -548,7 +609,27 @@ def get_group_names_from_link_name(self, link_name):
         return group_names
 
     def get_position_by_joint_name(self, configuration, joint_name, group=None):
-        """Returns the value of the joint_name in the passed configuration.
+        """Returns the joint value of the joint with ``joint_name`` in the passed configuration.
```

Comment:
Lots of `joint` redundancy in that sentence.

Reply:
Hm yeah, I just found it a bit weird to say `value of the joint_name`.

Maybe: "Returns the position of a joint in a given configuration."?

Reply:
Yep, sounds better